### ISSUE TITLE
When editing a file larger than about 10k words the word count goes to 0 words until I recalculate.

### ISSUE THREAD

**Issue by tyson-nw** [isaaclyman/novel-word-count-obsidian#66]
**Description**

I recalculate and it shows 13,722 words. As soon as I start typing it goes to 0 words. 

**Device**

This is happening on a WIndows machine running Obsidian 1.4.16

**Console Log**
```
novel-word-count: [changed] metadataCache hook fired, recounting file Campaign/Queen's Hand.md
plugin:novel-word-count:43 novel-word-count: Setting display counts for 3 fileItems matching path Campaign/Queen's Hand.md
plugin:novel-word-count:58 novel-word-count|updateDisplayedCounts [Campaign/Queen's Hand.md]: 0.56298828125 ms
plugin:novel-word-count:43 novel-word-count: [changed] metadataCache hook fired, recounting file Campaign/Queen's Hand.md
plugin:novel-word-count:43 novel-word-count: Setting display counts for 3 fileItems matching path Campaign/Queen's Hand.md
plugin:novel-word-count:58 novel-word-count|updateDisplayedCounts [Campaign/Queen's Hand.md]: 0.5419921875 ms
plugin:novel-word-count:43 novel-word-count: [changed] metadataCache hook fired, recounting file Campaign/Queen's Hand.md
plugin:novel-word-count:43 novel-word-count: Setting display counts for 3 fileItems matching path Campaign/Queen's Hand.md
plugin:novel-word-count:58 novel-word-count|updateDisplayedCounts [Campaign/Queen's Hand.md]: 0.531005859375 ms
```

**Comment by isaaclyman** [comment]
Hi @tyson-nw, can you confirm you're on the latest version of the plugin (2.25.1)?

**Comment by tyson-nw** [comment]
Yep. The file is also in a subfolder. Here is the file I am having an issue with.

[Queen's Hand.md](https://github.com/isaaclyman/novel-word-count-obsidian/files/13357044/Queen.s.Hand.md)

**Comment by isaaclyman** [comment]
Thanks for providing that, I was able to reproduce the issue on that file immediately.

Fixed in v2.25.2.
